src/components/ProductSearchOverlayResult.vue

```diff
@@ -31,6 +31,8 @@ const emit = defineEmits(['search', 'update:sort', 'update:page', 'search-for'])
 const sortValue = ref(props.sort);
 const pageValue = ref(props.page);
 
+const contentResults = computed(() => props.contentRecommendationResult?.results?.slice(0, 10));
```

Comment:
Why is the `slice` introduced here?

Reply:
To reduce the number of API calls, we did 2 content searches, but for 1 we only want to show 10, so to simply it a bit I do a slice here to keep it to 10 results